# Gene Regulation Research Projects 
## 1. Impact of Smoking on Epigenetic Mechanisms in Lung Cancer Susceptibility  
**Authors:** M. Golchinpour, A. Fotuhi Siahpirani, Z. Salehi  
**Conference:** Presented at the 4th International Bioinformatics Conference (ICB13), Zanjan, Iran  
**Description:**  
To assess how smoking, as an environmental factor, affects lung cancer susceptibility, we reanalyzed a publicly available multiome dataset (GSE241468) profiling transcriptome and chromatin accessibility in 117,911 lung cells from smokers and never-smokers. Our analysis revealed differences in chromatin accessibility and gene expression, along with enrichment of transcription factor binding motifs in smoking-responsive regions, highlighting cell-type-specific elements linked to increased cancer risk.  
ðŸ“„ [Poster PDF](https://github.com/mariyagolchin/projects/blob/main/Poster_golchinpour.pdf)

---

## 2. Cooperative Transcription Factor Binding in Ovarian Aging  
**Authors:** M. Golchinpour, A. Fotuhi Siahpirani, Z. Salehi  
**Conference:** 25th Royan International Congress on Reproductive Biomedicine, Tehran, Iran (submitted)  
**Description:**  
We analyzed single-nuclei multi-omics data (GSE202601) from young and aged human ovaries to identify cooperative transcription factors regulating gene expression in granulosa cells. The study mapped 7,262 differentially accessible regions and 4,514 target genes, followed by motif scanning using 959 PWMs. Enrichment analysis revealed over 180,000 significant TF co-binding interactions, providing a detailed map of combinatorial TF activity in ovarian aging.

---

## 3. Review Article: Computational Approaches for Transcription Factor Binding Site Identification  
**Authors:** M. Golchinpour, M. Dousti, F. Reisi, L. Shafaghati, Z. Bayat, Z. Salehi, A. Fotuhi Siahpirani  
**Status:** In progress  
**Description:**  
Transcription factors play a key role in gene regulation, yet identifying their exact binding sites (TFBSs) remains challenging. This review compiles key studies on TFBS prediction, covering both traditional and modern computational methods, with a special focus on deep learning techniques.

---

## 4. TFBS-Prediction  
**Authors:** M. Golchinpour, M. Dousti, N. Esfehani, A. Fotuhi Siahpirani  
**Status:** In progress  
**Description:**  
This project implements computational strategies for TFBS prediction using deep learning and motif-based approaches. Based on ChIP-seq and DNase-seq data from K562 and GM12878 cell lines, we generated training sets and applied models like DanQ to evaluate TFBS prediction performance.  
ðŸ”— [Private Project Link](https://github.com/mariyagolchin/TFBS-Prediction)


# Bioinformatics Coursework Projects

- **[Mathematical Foundations of Bioinformatics in R](https://github.com/mariyagolchin/MFB-R-Programming-1/blob/main/HW_P01.pdf)**  
  Supervisor: Dr. Hesam Montazeri - Dr.Alireza Fotuhi Siahpirani
- **[Developing an R Function from Scratch for Oncoplot Generation and Comparison with maftools](https://github.com/mariyagolchin/MFB-R-Programming-2/blob/main/HW_P02.pdf)**  
  Supervisor: Dr. Hesam Montazeri - Dr.Alireza Fotuhi Siahpirani
- **[TCGA Data Analysis and Uniform Distribution in R](https://github.com/mariyagolchin/MFB-R-Programming-4/blob/main/HW_P04.pdf)**  
  Supervisor: Dr. Hesam Montazeri  - Dr.Alireza Fotuhi Siahpirani 

- **[Read Mapping and Genome Assembly](https://github.com/mariyagolchin/Read-mapping-and-genome-assembly/blob/main/Project_questions.pdf)**  
  - Part A: Downloading, Analyzing, and Quality Control of *E. coli* Whole Genome Sequencing (WGS) Data  
  - Part B: De novo Genome Assembly  
  - Part C: Read Mapping and Variant Calling  
  Supervisor: Dr. Hesam Montazeri - Dr.Alireza Fotuhi Siahpirani 
- **[RNA-Seq Analysis](https://github.com/mariyagolchin/RNA-Seq-analysis/blob/main/AIB_proj2_1402.pdf)**  
  - Part A: Quality Control and Read Trimming  
  - Part B: Read Mapping to Reference Genome  
  - Part C: Constructing Gene Expression Matrix  
  - Part D: Differential Gene Expression Analysis  
  - Part E: Gene Ontology (GO) Enrichment Analysis  
  Supervisor: Dr. Hesam Montazeri  - Dr.Alireza Fotuhi Siahpirani 
- **[Learning Phylogenetic Trees from Multiple Sequence Alignment - COVID Data](https://github.com/mariyagolchin/Learning-Phylogenetic-Trees/blob/main/Project_questions.pdf)**  
  - Part A: Data Preparation  
  - Part B: Phylogenetic Analysis  
  - Part C: Data Visualization  
  Supervisor: Dr. Hesam Montazeri   - Dr.Alireza Fotuhi Siahpirani

- **[Predicting pIC50 for Alzheimerâ€™s Disease using Machine Learning and ChEMBL](https://github.com/mariyagolchin/Predicting-PIC50-with-machine-learning)**  
  Supervisor: Dr. Sajjad Gharaghani  

- **[Design and Implementation of a Bioinformatics Web Portal Inspired by cBioPortal Using Django](https://github.com/mariyagolchin/Cbioportal)**  
  Supervisor: Dr. Kaveh Kavousi, TA: Karim Rahimian  



# Research Projects Pursued Out of Personal Interest

- **[Analyzing the Adult Mouse Brain by scATAC-seq Data](https://github.com/mariyagolchin/Analyzing-adult-mouse-brain-scATAC-seq-Compiled-August-20-2024?tab=readme-ov-file)**  
- **[Epigenetic Traits Reflected in Chromatin Accessibility of Aged Hematopoietic Stem Cells Analyzed by scATAC-seq Using R](https://github.com/mariyagolchin/implementing-Epigenetic-traits-inscribed-in-chromatin-accessibility-in-aged-hematopoietic-stem-cells/tree/main)**  
- **[Labeling and Clustering of scRNA-seq Data Using Seurat and SingleR](https://github.com/mariyagolchin/LableCells_SingleR)**  
- **[Inference of Gene Regulatory Networks from Expression Data Using GENIE3](https://github.com/mariyagolchin/GENIE3_GRN-Inference)**  
- **[Single Cell RNA-seq Analysis in R Using the Seurat Package](https://github.com/mariyagolchin/scRNA-seq-analysis-with-Seurat)**  
- **[RNA-Seq Analysis Using DESeq2](https://github.com/mariyagolchin/RNAseq_analysis)**  
- **[A Primer on Deep Learning in Genomics](https://github.com/mariyagolchin/A-Primer-on-Deep-Learning-in-Genomics)**  
- **[Genomic Data Science - Biopython](https://github.com/mariyagolchin/Python-for-Genomic-Data-Science/blob/main/Python_for_Genomic_Data_Science.ipynb)**  
- **[Breast Cancer Classification Using a Support Vector Machine](https://github.com/mariyagolchin/Breast-Cancer-Classification-Using-a-Support-Vector-Machine?tab=readme-ov-file)**  
- **[Algorithms for DNA Sequencing by Johns Hopkins University](https://github.com/mariyagolchin/Algorithms-for-DNA-Sequencing-Ben-Langmead)**  
- **[Python and Bioinformatics](https://github.com/mariyagolchin/Python-and-Bioinformatics)**  
- **[Intermediate Pandas: Python Library for Data Science](https://github.com/mariyagolchin/IntermediatePandasPythonLibraryforDataScience)**  
- **[Deep Learning in Keras](https://github.com/mariyagolchin/Deep-Learning-in-Keras-Tutorial)**


# Specialized Courses

Please check my full list of certifications on my [LinkedIn Certifications](https://www.linkedin.com/in/mariyagolchinpour/details/certifications/) page.

- **Introduction to Next-Generation Sequencing (NGS)** â€“ Tehran University of Medical Sciences, Immunology Student Association  
- **Algorithms for DNA Sequencing** â€“ Johns Hopkins University (Coursera), Grade: 91.42%
- **Introduction to Genomic Technologies** â€“ Johns Hopkins University (Coursera), Grade: 82.50%
- **Machine Learning Models in Science** â€“ LearnQuest (Coursera), Grade: 94%
- **Python for Genomic Data Science** â€“ Johns Hopkins University (Coursera), Grade: 82.40%
- **Introduction to Data Science and scikit-learn in Python** â€“ LearnQuest (Coursera), Grade: 93.37%
- **Supervised Machine Learning: Regression and Classification** â€“ DeepLearning.AI, Stanford University, Grade: 100%

